src/main/java/com/qudini/api/RequestSender.java

```diff
@@ -0,0 +1,215 @@
+package com.qudini.api;
```

Comment:
This class has been only migrated, it has no changes except for the class imports and configurations that would break the build